tests/Feature/SwooleProfilerRegisteredTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace K911\Swoole\Tests\Feature;
+
+use K911\Swoole\Bridge\Upscale\Blackfire\WithProfiler;
+use K911\Swoole\Server\Configurator\CallableChainConfigurator;
+use K911\Swoole\Server\Configurator\WithRequestHandler;
+use K911\Swoole\Tests\Fixtures\Symfony\TestBundle\Test\ServerTestCase;
+use ReflectionClass;
+
+final class SwooleProfilerRegisteredTest extends ServerTestCase
+{
+    /**
+     * Ensure that WithProfiler is added after WithRequestHandler.
+     */
+    public function testWiring(): void
```

Comment:
I do not think we need to test this deeply. We should only check whether blackfire profiler and configurator has been registered in container. The ordering of configurations could be a bug, that needs to be fixed in separate PR, and will probably disappear after the refactor (different approach to configure events and swoole server - branch `feat/dynamic-listeners`).

Reply:
I don't understand this, what do you mean that the ordering could be a bug? The WithRequestHandler is added manually to the list of configurators, not via autoconfiguration, so the ordering seems to be fixed to me (if I autoconfigure WithProfiler instead of adding it manually, it's always triggered earlier).

I had to ensure that the profiler was enabled after registering the request handler because otherwise blackfire would throw an exception related to missing middleware or something (can't remember at the moment sorry), that's why I was testing for this, but I can remove it if you want

Reply:
> I had to ensure that the profiler was enabled after registering the request handler because otherwise blackfire would throw an exception related to missing middleware or something (can't remember at the moment sorry), that's why I was testing for this, but I can remove it if you want

This part you're describing is a probably buggy behaviour in swoole-bundle itself, so instead of making sure the ordering of configurators is right in this test, we should focus on fixing that issue or more like understanding it first - but it is not the purpose of this PR.

I'll comment on lines which I think are unnecessary

Reply:
Ok now I understood, thanks!